src/migrations/env.py

```diff
@@ -1,23 +1,20 @@
 import asyncio
 from logging.config import fileConfig
 
+from alembic import context
+from app.core.config import settings
+from app.core.db.database import Base
 from sqlalchemy import pool
 from sqlalchemy.engine import Connection
 from sqlalchemy.ext.asyncio import async_engine_from_config
 
-from alembic import context
-
-from app.core.db.database import Base
-from app.core.config import settings
-from app.models.user import *
```

Comment:
@igorbenav mind checking this change? not sure what we are importing here and ruff doesn't allow star imports

Reply:
It's working fine